frontend/factory-ops/src/component/ContentPicker.jsx

```diff
@@ -6,12 +6,24 @@ export default function ContentPicker({ onViewChange }) {
 	};
 
 	return (
-		<div>
-			<button className="btn btn-primary btn-lg btn-block m-3" onClick={() => handleButtonClick('PLAN')}>
+		<div className="btn-group-vertical mt-5">
+			<button className="btn btn-outline-primary btn-lg" onClick={() => handleButtonClick('PLAN')}>
 				PLAN
 			</button>
-			<button className="btn btn-primary btn-lg btn-block m-3" onClick={() => handleButtonClick('ITEM')}>
-				ItemList
+			<button className="btn btn-outline-primary btn-lg" onClick={() => handleButtonClick('ITEM')}>
```

Comment:
1. Why do we call it 'PLAN' in one instance and 'ITEM' in another? For me, 'ItemList' should gather all the data and filter it internally if needed."

2. For MVP purposes, we don't need plans split into machine groups. All machines can be part of one large plan.